tests/test_stl_binders.py

```diff
@@ -67,6 +67,7 @@ def test_vector_int():
     v_int2.clear()
     assert len(v_int2) == 0
 
+
```

Comment:
Accident?

Reply:
Yes. Still in progress, playing with fixing PyPy. Will fix!

Reply:
Actually, no, it was a flake8 request.